### DISCUSSION TITLE
2023 Control System is out!

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#236] [Team Posts]
There will be several iterations, but the first non-beta release is out.
https://github.com/wpilibsuite/allwpilib/releases

The 2023.1.1 release is the first. We will want to track that releases page for updates as we approach our first comp.

-Chris

**Comment by coverbeek** [comment]
2023.2.1 was just released. In VSCode go to the WPILib command palette and select "Check for WPILib updates" and update. We will do the coding computers and the competition computer(s) when I am present Monday evening.

**Comment by programmeruser517** [comment]
Just got updated to 3.2. Didn't know if this was announced in person already, so I figured I'd put it here. **All pull requests** should probably reflect this update before even being created (via updating all your computers' WPILib).
